packages/daimo-api/src/server/server.ts

```diff
@@ -44,6 +45,10 @@ async function main() {
   const accountFactory = new AccountFactory(vc);
   const crontab = new Crontab(vc, coinIndexer, nameReg, monitor);
 
+  const shovelWatcher = new Watcher();
+  shovelWatcher.addCallback(coinIndexer.load);
+  setInterval(() => shovelWatcher.run(), 1000);
```

Comment:
nice
so watcher polls, and then:
- poll latest block from pg where we know that all the logs from that block are already ingested
- if > current latest, call all callbacks passing (current, latest)
- each of those goes to an indexer, which loads the logs it needs in that range & processes them
- since we register the indexers in a specific order, the ones that depend on other indexers stay happy

Reply:
once we have it dialed in, poll interval can be much faster